docs/src/tutorials/tutorial_1.md

```diff
@@ -4,4 +4,336 @@
 
 If you are reading this tutorial, I am going to assume this is the first time you are using `Javis` to create an animation. In that case, welcome to `Javis`! 😃 By following this tutorial, we are going to make you a director of your very own animations written in pure Julia! 🎬 🎥
 
-If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](../index.html#Installation).
+If you have not installed `Javis` yet, please visit the homepage to [read the installation instructions](/index.html#Installation).
+
+## Explanation
+
+Luxor.jl drawing onto a canvas. It provides simple functions like line, circle and so on and does animation. If you're interested in 2D graphics, you should definitely check out the awesome Luxor Tutorial.
```

Comment:
For functions you might want to use the `CODE` syntax.

Reply:
and the first sentence has some weird grammar. It is also a good point to add a link to https://github.com/JuliaGraphics/Luxor.jl at that point.